Cargo.toml

```diff
@@ -13,8 +13,8 @@ repository = "https://github.com/sassman/ssd-benchmark-rs"
 exclude = [".ci/*", ".gitignore", ".travis.yml"]
 
 [dependencies]
-rand = "0.7"
-figlet-rs = "0.1"
+rand = "0.7.3"
```

Comment:
Please keep the 2 dependencies as they are, pinned without the minor version number. That is intentionally, to avoid upgrading on minor upstream changes.